specifyweb/frontend/js_src/lib/components/WbImport/helpers.ts

```diff
@@ -65,7 +65,7 @@ export function extractHeader(
   hasHeader: boolean
 ): { readonly rows: RA<RA<string>>; readonly header: RA<string> } {
   const header = hasHeader
-    ? uniquifyHeaders(data[0].map(f.trim))
+    ? uniquifyHeaders(data[0].map((value) => f.trim(value.toString())))
```

Comment:
what was the type of the value on which the code tried to call .trim() and failed?

was it undefined, or null or number? whatever it was, the typescript type shows that the type would always be string - so at very least you should modify the typescript type on line 64
but you should go one step further and see what caused some non-string values to be there in the first place

Reply:
it was number